scripts/config/msu/classes/enum.nut

```diff
@@ -0,0 +1,49 @@
+::MSU.Class.Enum <- class
+{
+	_Keys = null;
+	_ValueKeyArray = null;
+
+	constructor( _array = null )
+	{
+		this._Keys = {};
+		this._ValueKeyArray = [];
+		if (_array != null) this.addArray(_array);
+	}
+
+	function _isValidKey( _key )
```

Comment:
Private functions should be prefixed with `__` i.e. two underscores. This is to differentiate them from metamethods.

Reply:
That's fair